runtime/basilisk/src/lib.rs

```diff
@@ -138,7 +138,6 @@ impl Contains<Call> for BaseFilter {
 
 		#[allow(clippy::match_like_matches_macro)]
 		match call {
-			Call::XYK(_) => false,
```

Comment:
Just for my understanding: This is basically the filter we were talking about, where we can disable features for our runtime? So in this case, we enable XYK as this line is removed...

Reply:
yes